packages/provider/src/KnownRelaysManager.ts

```diff
@@ -44,6 +44,8 @@ export class KnownRelaysManager {
   private readonly scoreCalculator: AsyncScoreCalculator
 
   private latestScannedBlock: number = 0
+  cachedLatestEvents: EventData[] = []
```

Comment:
Caching is dangerous so I suggest not doing it 1 day before release
Lets just look for registration events for a way smaller window